approval_polls/views.py

```diff
@@ -30,12 +33,19 @@ def getPolls(request, poll_list, render_page):
       polls = paginator.page(paginator.num_pages)
   return render(request, render_page, {'latest_poll_list' : polls})
 
-class DetailView(generic.DetailView):
+class DetailView(generic.DeleteView):
   model = Poll
   template_name = 'approval_polls/detail.html'
   def get_queryset(self):
       return Poll.objects.filter(pub_date__lte=timezone.now())
 
+  @staticmethod
```

Comment:
@ashimaathri Actually I am a little bit confused now.
The delete method is under DetailView, but the delete button is not under the detail page, so I am not sure how I can delete the object under my poll view calling method under detailview.
Could you elaborate on it or  provide me some other example?
Thanks!